pkg/clusteragent/orchestrator/controller.go

```diff
@@ -8,11 +8,16 @@
 package orchestrator
 
 import (
+	"fmt"
 	"math/rand"
 	"net/http"
 	"sync/atomic"
 	"time"
 
+	model "github.com/DataDog/agent-payload/process"
+
+	"github.com/DataDog/datadog-agent/pkg/forwarder"
```

Comment:
nit: group this up with import below

Reply:
Hmm, this is how go imports formats them for me.   Does it do something different for you?

Reply:
in the agent we usually have 3 groups for imports
- native
- project
- 3rd party

IIRC, if you group them up with the ones below they should stick to the group and be ordered alphabetically